docs/install_guides/installing-openvino-yum.md

```diff
@@ -1,10 +1,210 @@
+# Install OpenVINO™ Runtime on Linux Using YUM Repository {#openvino_docs_install_guides_installing_openvino_yum}
 
-Currently only the following ways are provided to install OpenVINO™:
+This guide provides installation steps for OpenVINO™ Runtime for Linux distributed through the YUM repository.
 
-* [Install OpenVINO Runtime from an Archive File](installing-openvino-from-archive-linux.md)
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. If you want to develop or optimize your models with OpenVINO, see [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
 
-The other installation methods are temporarily unavailable.
\ No newline at end of file
+> **IMPORTANT**: By downloading and using this container and the included software, you agree to the terms and conditions of the [software license agreements](https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf). Please review the content inside the `<INSTALL_DIR>/licensing` folder for more details.
+
+## System Requirements
+
+The complete list of supported hardware is available in the [Release Notes](https://software.intel.com/content/www/us/en/develop/articles/openvino-relnotes.html).
+
+**Operating systems**
+
+- Red Hat Enterprise Linux 8, 64-bit
+
+**Software**
+
+- [CMake 3.13 or higher, 64-bit](https://cmake.org/download/)
+- GCC 8.2.0
+- [Python 3.7 - 3.10, 64-bit](https://www.python.org/downloads/windows/)
+
+## Install OpenVINO Runtime
+
+### Step 1: Set Up the Repository
+
+1. Create the YUM repo file in the `/tmp` directory as a normal user:
+   ```
+   tee > /tmp/openvino-2022.repo << EOF
+   [OpenVINO]
+   name=Intel(R) Distribution of OpenVINO 2022
+   baseurl=https://yum.repos.intel.com/openvino/2022
+   enabled=1
+   gpgcheck=1
+   repo_gpgcheck=1
+   gpgkey=https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+   EOF
+   ```
+2.	Move the new openvino-2022.repo file to the YUM configuration directory `/etc/yum.repos.d`:
+   ```sh
+   sudo mv /tmp/openvino-2022.repo /etc/yum.repos.d
+   ```
+3.	Verify that the new repo is properly setup by running the following command:
+   ```sh
+   yum repolist | grep -i openvino
+   ```
+    You will see the available list of packages.
+
+
+To list available OpenVINO packages, use the following command:
+
+@sphinxdirective
+
+   .. code-block:: sh
+
+      yum list 'openvino*'
+
+@endsphinxdirective
+
+### Step 2: Install OpenVINO Runtime Using the YUM Package Manager
+
+OpenVINO™ Runtime will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<PATCH>`
+
+A symlink will be created: `/opt/intel/openvino_<VERSION>`
```

Comment:
packages are installed to the system locations, not in /opt

Reply:
Shall we specify here, which system locations they are?

Reply:
We will specify the proper directories in this article, once we have had substantial information. For now, I have removed these two lines with https://github.com/openvinotoolkit/openvino/pull/14492/commits/5fbef2ce42dd812b17f51e622fa15c1284a140cf